### DISCUSSION TITLE
Memory is overutilized, so cooling down segment

### DISCUSSION THREAD

**Discussion by vlaterz** [datalust/seq-tickets#1652] [Help]
Seq log wrote these entries and then seq died.
![image](https://user-images.githubusercontent.com/33147329/188861432-44dac482-2384-49b8-b2d8-a4b5660d5c64.png)
![image](https://user-images.githubusercontent.com/33147329/188861461-9aa5b601-ae28-4052-98d5-668f9e34b764.png)
Seq is in docker
Task manager said that server has 50gb of RAM, 30Gb of RAM is used, and 16GB is cached.   
What does mean:
"Memory is overutilized, so cooling down segment "2022-08-21T12:00:00.0000000Z (95261 events)" ";
"Metrics sampled";
"Native storage metrics sampled";
"Out of memory"
and why SEQ stops?

**Comment by nblumhardt** [comment]
Hi Vladimir, thanks for reaching out.

"Memory is overutilized" is Seq functioning normally; Seq caches events in RAM as they arrive, and once it detects memory pressure, drops the oldest events out. The message will be printed for every one hour cache segment that is dropped.

The messages relating to metrics are also normal. Those events carry additional properties not shown in the text log output, such as memory usage, cache information, ingestion rate, etc.

It's not clear how the container has ended up OOMing, though those metrics should help with troubleshooting. They're included in the diagnostic report, which you can download from Settings > Diagnostics > Download a full diagnostic report, sanitize if necessary, and send to us via `support@datalust.co`.

If you can also include a screenshot of the All Time chart from Data > Storage, that would be great!

Best regards,
Nick

**Comment by vlaterz** [comment]
We cannot get logs from ettings > Diagnostics > Download a full diagnostic report, because we can`t even open a website. Is there any folder from which we can take logs?

**Reply by nblumhardt** [reply]
Yes - in the container's mounted data volume you'll find a `Logs` folder with JSON text files.

Just to double-check - have you tried restarting the container? (It seems strange it'd OOM before you're able to reach the diagnostics page :thinking:.)
